Mathlib/Topology/Algebra/Group/Basic.lean

```diff
@@ -1216,12 +1216,40 @@ namespace Units
 
 open MulOpposite (continuous_op continuous_unop)
 
+@[to_additive]
+theorem range_embedProduct [Monoid α] :
```

Comment:
I don't know what's the right place to put this: `Set.range` doesn't exist where `embedProduct` is defined...